concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -235,7 +235,8 @@ resources:
       bucket: ((state_bucket))
       region_name: ((aws_region))
       versioned_file: bosh-vars-store.yml
-      initial_version: "{}"
+      initial_version: "-"
+      initial_content_text: "{}"
```

Comment:
I think this line is not needed, especially, that this is a yaml file and not a json

Reply:
We talked about this and there is a migration script which doesn't handle an empty yaml.